src/async/imap/MCIMAPFetchContentToFileOperation.h

```diff
@@ -0,0 +1,47 @@
+//
+//  IMAPFetchContentToFileOperation.h
+//  mailcore2
+//
+//  Created by Dmitry Isaikin on 2/08/16.
+//  Copyright (c) 2016 MailCore. All rights reserved.
+//
+
+#ifndef MAILCORE_IMAPFETCHCONTENTTOFILEOPERATION_H
+
+#define MAILCORE_IMAPFETCHCONTENTTOFILEOPERATION_H
+
+#include <MailCore/MCIMAPOperation.h>
+
+#ifdef __cplusplus
+
+namespace mailcore {
+
+    class MAILCORE_EXPORT IMAPFetchContentToFileOperation : public IMAPOperation {
+    public:
+        IMAPFetchContentToFileOperation();
+        virtual ~IMAPFetchContentToFileOperation();
+
+        virtual void setUid(uint32_t uid);
+        virtual void setPartID(String * partID);
+        virtual void setEncoding(Encoding encoding);
+        virtual void setOutputFile(String * outputFile);
+        virtual void setChunkSize(uint32_t chunkSize);
```

Comment:
> For example, for Rambler IMAP server good value would be 100kb, for other servers is 2-5Mb.

Do you have other servers where just fetching would fail?
I'd like to cover the case of filling memory with the feature we talked about on libetpan.